src/charm.py

```diff
@@ -1153,14 +1181,86 @@ def _create_pgdata(self, container: Container):
         container.exec([
             "chown",
             f"{WORKLOAD_OS_USER}:{WORKLOAD_OS_GROUP}",
-            "/var/lib/postgresql/logs",
+            logs_path,
         ]).wait()
         container.exec([
             "chown",
             f"{WORKLOAD_OS_USER}:{WORKLOAD_OS_GROUP}",
-            "/var/lib/postgresql/temp",
+            temp_path,
         ]).wait()
 
+    def _fix_health_check_script(self, container: Container) -> None:
```

Comment:
This needs to be removed and instead we need to update the script in it's repository (https://github.com/canonical/charmed-postgresql-rock/blob/4dbbaa70461ee8977244ad0f322b274a600caa53/local/scripts/self-signed-checker.py).